collect_app/src/test/java/org/odk/collect/android/widgets/DrawWidgetTest.java

```diff
@@ -42,7 +42,8 @@
 @RunWith(RobolectricTestRunner.class)
 public class DrawWidgetTest extends FileWidgetTest<DrawWidget> {
 
-    private static final String ANSWER_TEXT = "jr://images/someURI";
+    private static final String DEFAULT_IMAGE_ANSWER = "jr://images/someURI";
```

Comment:
Could be local to `whenPromptHasDefaultAnswer_showsInImageView` but its functionality parallels `USER_SPECIFIED_IMAGE_ANSWER ` which has to be at least an instance variable.

Reply:
I like the symmetry as well.